simp_config_answers.yaml

```diff
@@ -66,47 +54,47 @@ gateway: "10.0.2.2"
 # convert your system to a caching DNS server. You'll know
 # that you have this situation if you end up with a duplicate
 # definition for File['/etc/named.conf'].
-"dns::servers":
-  - "192.168.33.10"
-  - "10.0.2.3"
-  - "8.8.8.8"
+dns::servers:
+  - '192.168.33.10'
```

Comment:
Is the named config recursive? I left the default vbox DNS server in there because I was unsure